Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +9,128 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            string personWithHighestBalanceEver = "not found";
+            double? highestBalance = null;
```

Comment:
Instead of making it nullable, use the smallest possible balance that there can be. For example - double.MaxValue. Like that, every number will be bigger than this one 😄

Reply:
oh, that a cool trick!